src/openzaak/components/documenten/api/serializers.py

```diff
@@ -607,59 +583,31 @@ def update(self, instance, validated_data):
         )
         validated_data["pk"] = None
         validated_data["versie"] += 1
-
-        if settings.CMIS_ENABLED:
-            # The fields integriteit and ondertekening are of "GegevensGroepType", so they need to be
-            # flattened before sending to the DMS
-            flat_integriteit = (
-                flatten_gegevens_groep(integriteit, "integriteit")
-                if integriteit
-                else {}
-            )
-            flat_ondertekening = (
-                flatten_gegevens_groep(ondertekening, "ondertekening")
-                if ondertekening
-                else {}
-            )
-            validated_data.update(**flat_integriteit, **flat_ondertekening)
-        else:
-            # Remove the lock from the data from which a new
-            # EnkelvoudigInformatieObject will be created, because lock is not a
-            # part of that model
-            validated_data.pop("lock")
+        validated_data.pop("lock")
```

Comment:
let's keep the comment
```suggestion
            # Remove the lock from the data from which a new
            # EnkelvoudigInformatieObject will be created, because lock is not a
            # part of that model
        validated_data.pop("lock")
```